### ISSUE TITLE
Link missing a text alternative

### ISSUE THREAD

**Issue by pooriamehregan** [Informasjonsforvaltning/fdk-issue-tracker#675]
[Link missing a text alternative - Issues - Accessibility NextGen - Accessibility.pdf](https://github.com/Informasjonsforvaltning/fdk-issue-tracker/files/10402235/Link.missing.a.text.alternative.-.Issues.-.Accessibility.NextGen.-.Accessibility.pdf)
